input.css

```diff
@@ -141,48 +152,34 @@ pre {
   text-align: right;
 }
 
-/* Style the data-playground search results. */
```

Comment:
Styles for the nav bar were re-organized so that they'd be together and easier to locate. The `data playground` styles were moved further down the file and not removed.